src/Conversions.cc

```diff
@@ -113,3 +114,22 @@ ignition::common::MouseEvent ignition::gui::convert(const QMouseEvent &_e)
   return event;
 }
 
+//////////////////////////////////////////////////
+ignition::common::KeyEvent ignition::gui::convert(const QKeyEvent &_e)
+{
+  common::KeyEvent event;
+  event.SetKey(_e.key());
+  event.SetText(_e.text().toStdString());
+
+  event.SetControl(
+    (_e.modifiers() & Qt::ControlModifier)
+    && (_e.key() == Qt::Key_Control));
```

Comment:
Do we need to also check `_e.key()`? I think this would only set control to true the moment the control key is pressed, but not when other keys are pressed together with control, which is what we'd like to capture here.